Makefile

```diff
@@ -197,70 +198,7 @@ $(Signed_RustEnclave_Name): $(RustEnclave_Name)
 	@echo "SIGN =>  $@"
 	@echo
 	@echo "Enclave is in $(SGX_ENCLAVE_MODE)"
-.PHONY: benchmark
-benchmark: $(Client_Name) $(Worker_Name) $(Signed_RustEnclave_Name)
```

Comment:
Keep target benchmark and print an error message about new syntax

Reply:
Hm, is this really necessary? We have not yet merged anything to the develop branch, so `make benchmark` itself would be a new syntax.

Reply:
Ah, please note, this is a PR to the branch of @zktony , not to develop.